Default (OSX).sublime-settings

```diff
@@ -0,0 +1,4 @@
+{
+  "use_local_url": false,
+  "local_url": "file:///Applications/Unity/Documentation/en/ScriptReference/30_search.html"
```

Comment:
On Yosemite and Unity 4.6 I'm finding the documentation locally at `file:///Applications/Unity/Documentation/html/en/ScriptReference/30_search.html`. Is this a typo or does yours differ?